fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -249,23 +249,21 @@ public FHIRResource() throws Exception {
     public Response metadata() throws ClassNotFoundException {
         log.entering(this.getClass().getName(), "metadata()");
         Date startTime = new Date();
-        Response.Status status = Response.Status.INTERNAL_SERVER_ERROR;
         String errMsg = "Caught exception while processing 'metadata' request.";
 
         try {
             checkInitComplete();
 
-            status = Response.Status.OK;
-            RestAuditLogger.logMetadata(httpServletRequest, startTime, new Date(), status);
+            CapabilityStatement capabilityStatement = getCapabilityStatement();
+            RestAuditLogger.logMetadata(httpServletRequest, startTime, new Date(), Response.Status.OK);
 
-            return Response.ok().entity(getCapabilityStatement()).build();
-        } catch (FHIRHttpException e) {
+            return Response.ok().entity(capabilityStatement).build();
```

Comment:
do you need the capabilityStatement instance? immediately responding with it seems preferred

Reply:
I forget why I split it out...maybe I thought I was going to use it but then never did.  I can revert that part of the change and call it inline if you prefer.

Reply:
In general, I'd love to start caching this, maybe on a per-tenant basis....our calls to `metadata` take a long time and it shouldn't change unless someone updates the config...

Reply:
no need to revert, I wanted to confirm the intent.  I didn't know if there was a side effect